src/Api/follow.ts

```diff
@@ -0,0 +1,34 @@
+import api from '@/Api/api';
+import { getAxiosHeader } from './user';
+
+const getFollowUser = async (userId: string) => {
+  try {
+    const headers = getAxiosHeader();
+    if (!headers) return false;
+    const followResponse = await api.post('/follow/create',
```

Comment:
p5;
```suggestion
    const {data} = await api.post('/follow/create',
```
이렇게 하고 바로 return data하는것도 좋은 방법인것같아요~

Reply:
data.. destructuring? 하겠습니다